index.d.ts

```diff
@@ -1,7 +1,4 @@
-interface Point {
-	x: number;
-	y: number;
-}
+type Point = [number, number];
```

Comment:
It's likely obvious from the context of what this tool is designed to do, but all the same it would be nice to mention in a comment here that `[number, number]` is representative of an x,y coordinate pair

Reply:
Good call— your comment made me recall I could type this as `[x: number, y: number]` so I did that instead!